src/dappsExplorer/search/NoResults.tsx

```diff
@@ -17,17 +17,23 @@ function NoResults({ testID, searchTerm }: Props) {
       <View style={styles.iconContainer}>
         <InfoIcon color={Colors.onboardingBlue} />
       </View>
-      <Text style={styles.text}>
-        <Trans i18nKey="dappsScreen.emptyResults.searchMessage" tOptions={{ searchTerm }}>
-          <Text style={styles.searchedText} />
-        </Trans>
-      </Text>
+      <View style={styles.searchTextContainer}>
+        <Text style={styles.text}>
+          <Trans i18nKey="dappsScreen.emptyResults.searchMessage" tOptions={{ searchTerm }}>
+            <Text style={styles.searchedText} />
+          </Trans>
+        </Text>
+      </View>
     </View>
   )
 }
 
 const styles = StyleSheet.create({
+  /* Using a 1:11 flex on two elements will create a pseudo 12 column layout */
```

Comment:
lol do we need this comment? 🤣 we're not using flex 11 anymore, so it might be confusing why we need to know this